plugins/gazebo_gui/GazeboGui.cc

```diff
@@ -47,23 +47,45 @@ bool GazeboGui::Load(const tinyxml2::XMLElement *_elem)
 
   // Set default config file for Launch
   std::string defaultConfigPath;
-  gz::common::env(IGN_HOMEDIR, defaultConfigPath);
+  gz::common::env(GZ_HOMEDIR, defaultConfigPath);
   defaultConfigPath = gz::common::joinPaths(defaultConfigPath,
-      ".ignition", "launch");
+      ".gz", "launch");
 
   auto defaultConfigFile = gz::common::joinPaths(defaultConfigPath,
       "gui.config");
 
+  // TODO(CH3): Deprecated. Remove on tock.
+  std::string defaultConfigPathDeprecated;
+  defaultConfigPathDeprecated = gz::common::joinPaths(defaultConfigPath,
```

Comment:
By the time we get here, `defaultConfigPath` won't just be `$HOME` anymore. So I think we'd need to move things around.

---

But in any case, I think we don't need to tick-tock the config file location. Otherwise, it's unclear which file is loaded, is it the one under `.ignition` or the one under `.gz`?

So we can hard-tock this and mention it in the migration guide.

Reply:
https://github.com/gazebosim/gz-launch/pull/166/commits/51407cdcdd8be27ff11b487dccb3ce7192a791ce hard-tocked!